.github/workflows/build.yml

```diff
@@ -186,6 +186,7 @@ jobs:
       - uses: DeterminateSystems/determinate-nix-action@main
       - uses: DeterminateSystems/flakehub-cache-action@main
       - name: Build manual
+        if: inputs.system == 'x86_64-linux'
         run: nix build .#hydraJobs.manual
       - uses: nwtgck/actions-netlify@v3.0
         if: inputs.publish_manual
```

Comment:
Probably want to limit this to  x86_64-linux as well, otherwise ./result/share/doc/nix/manual probably won't exist